docssrc/src/optional_arguments.md

```diff
@@ -78,24 +78,19 @@ However, calling `withOptionalArguments` is safer because it makes sure that the
 
 ## Avoiding null values
 
-Previously, we've looked at how to handle null values. To make all of this easier, the CommandAPI implements multiple `getOptional()` methods for `CommandArguments`:
-
-<div class="warning">
-
-> **Developer's Note:**
->
-> For 9.0.1, all `CommandArguments#getOrDefault()` methods have been deprecated and new methods have been added!
-> The existing methods will be removed in an upcoming version!
->
-> View the new methods below:
-
-</div>
+Previously, we've looked at how to handle null values. To make all of this easier, the CommandAPI implements multiple additional methods for [`CommandArguments`](./commandarguments.md):
 
 ```java
+Object getOrDefault(int index, Object defaultValue);
+Object getOrDefault(int index, Supplier<?> defaultValue);
+Object getOrDefault(String nodeName, Object defaultValue);
+Object getOrDefault(String nodeName, Supplier<?> defaultValue);
 Optional<Object> getOptional(int index)
 Optional<Object> getOptional(String nodeName)
 ```
 
+The examples will be using the `getOptional` methods but there is no downside of using the `getOrDefault` methods.
```

Comment:
Nice to mention this, but I think it's worth stating what the `getOrDefault` methods _do_ instead of brushing them aside with no explanation.

Reply:
Wouldn't it make more sense to only mention the `getOrDefault` here and explain it in the `CommandArguments` page?
That page is the first one in the `Command arguments` section anyway, so only mentioning it here is probably fine?

Reply:
> That page is the first one in the `Command arguments` section anyway

Good point.